demo/meeting/README.MD

```diff
@@ -12,11 +12,9 @@ This is the SDK react meeting demo app. It shows how to use the Amazon Chime SDK
 
 4. Install dependencies: `npm install`
 
-5. Start the webpack server: `npm run start:client`
+5. Start the webpack server and node server concurrently: `npm run start`
 
-6. Open a second terminal. Within the same directory, start the server: `npm run start:backend`
-
-7. Open https://0.0.0.0:9000/ in your browser
+6. Open https://0.0.0.0:8080/ in your browser
```

Comment:
When I run npm run start, and navigate to https://0.0.0.0:8080/, I dont get anything. If I go to http://127.0.0.1:8080/, I see the meeting demo

Reply:
Strange, I can access it by `https://0.0.0.0:8080/`, `https://127.0.0.1:8080/` and `https://localhost:8080/`

Reply:
Well, we are proxying the backend server calls to node server running on port 8080 and the UI is served from port 9000. This is useful as we have a separate API backend development server and we want to send API requests on the same domain as said [here](https://webpack.js.org/configuration/dev-server/#devserverproxy). Further, this works well with the serverless deployment we have. Can we keep the webpack config as is and make the addition for concurrent running?

Reply:
sounds good, revert webpack change. Thanks for the explanation.